src/schema/v2/OrderedSet/updateOrderedSetMutation.ts

```diff
@@ -43,7 +45,7 @@ interface Input {
 }
 
 const SuccessType = new GraphQLObjectType<any, ResolverContext>({
-  name: "updateOrderedSetSuccess",
+  name: "UpdateOrderedSetSuccess",
```

Comment:
We don't [use](https://github.com/search?q=org%3Aartsy+updateOrderedSetMutation&type=code) this type yet, so not breaking.

Reply:
Same for other similar typing.

Reply:
Actually we do (as of..this week?!). Just one mutation in Forque - https://github.com/artsy/forque/blob/53bf9acda74840e4d8441dd892025a62d599d018/src/pages/features/mutations/addOwnerToOrderedSet.tsx#L13

It's for 'adding sets to a feature' functionality. The way that's implemented is it's actually an update on the set, setting the `owner_id` to the feature id, and `owner_type: 'Feature'`.

Reply:
Ah, bad searching on my part. I've updated the typing in the related `forque` pr with [this commit](https://github.com/artsy/forque/pull/692/commits/18c7f339004f22a80826b78b4835ed7aca93a6e9).